auth/openid_connect.adoc

```diff
@@ -16,13 +16,14 @@ The current implementation only secures the Appliance's Web administrative UI wi
 
 *Note:* The REST API and Self-Service UI do not currently support OIDC.
```

Comment:
I guess this Note needs changing since REST API crossed the finish line.

Reply:
@abellotti Great Catch! I hadn't even noticed that! Thank you.